### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by scottgonzalez**

### CODE REVIEW

**Comment by jflewis** [comment]
Looks good to me 🚜

**Comment by kpollich** [comment]
This is awesome. Looks good to me.

File: example.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const irrigation = require('./irrigation')
```
Comment (scottgonzalez) [inline example.js:1]: Oops. I didn't mean to add this file. This code exists in the README. I just put it here to make sure it was formatted properly. I'll delete before merging.

**Comment by arschmitz** [comment]
👍

**Pull request closed without merging**